index.js

```diff
@@ -74,6 +74,20 @@ function parseIntoPreview(data) {
   });
 }
 
+function parseIntoTrackArray(data){
+  if(!data.name)
```

Comment:
the data object is going through the sanityCheck functino before this, so this check is not needed